components/brave_new_tab_ui/containers/newTab/gridPageButtons.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { useCallback, useRef, useEffect } from "react";
+import styled from "styled-components";
+
+const PageIndicator = <svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+    <circle cx="50" cy="50" r="50" />
+</svg>
+
+const ListPageButtonContainer = styled('div') <{}>`
+  display: flex;
+  flex-direction: row;
+
+  --list-page-button-size: 8px;
```

Comment:
Are there margin/sizing variables I should use here instead?

Reply:
Not yet, no. And maybe there won't ever be that apply to all UIs. It's fine to use specific values from the design for this one-off UI.